src/access_control.ruleset

```diff
@@ -106,99 +106,100 @@ ruleset access_control_ruleset
         }
     }
 
+    // Check if a non-stranger swiped their badge.
```

Comment:
This seems odd to me. Trying to connect it to real world workflow.
Presumably, a stranger does not have a badge to swipe.

Reply:
Fixed: `// Handles when someone swipes their badge.` The "non-stranger" part is likely hinting at the `!stranger` assertion in the rule. Which itself is probably paranoia, but this app isn't tested enough for us to know if we should be worried about strangers swiping badges.